modules/@ergonode/ui/src/components/AdvancedFilters/Content/AdvancedFilterContent.vue

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <div :class="classes">
```

Comment:
I think if wh have only one class name w should use:`class="advanced-filter-content"`.  Less code and a cleaner approach.